.pre-commit-config.yaml

```diff
@@ -7,3 +7,13 @@ repos:
   - id: check-yaml
   - id: mixed-line-ending
     args: [--fix=lf]
+- repo: https://github.com/psf/black
+  rev: 20.8b1
+  hooks:
+  - id: black
+    exclude: "migrations/"
```

Comment:
No need to exclude migrations, a lot of them contain hand written data migration code